src/components/encounter/ohri-encounter-form.component.tsx

```diff
@@ -416,7 +416,7 @@ export const OHRIEncounterForm: React.FC<OHRIEncounterFormProps> = ({
             location: encounterLocation,
             order: null,
             groupMembers: [],
-            uuid: field?.value?.uuid,
```

Comment:
@Herobiam just want to make sure this change does not affect other scenarios. Are all the tests OK? Unless this was always `null` for all the scenarios, otherwise I would add an if statement

Reply:
it was always null, I have also confirmed it with Jovan